_config/dynamiccache.yml

```diff
@@ -47,7 +47,7 @@ DynamicCache:
 # Determines which headers should also be cached. X-Include-CSS and other relevant
 # headers can be essential in instructing the front end to include specific
 # resource files
-  cacheHeaders: '/^(X\-)|(Content\-Type)|(Cache\-Control)|(Etag)|(Expires)|(Last\-Modified)|(Location)/i'
+  cacheHeaders: '/^(X\-)|(Content\-Type)|(Cache\-Control)|(Etag)|^(Expires)|(Last\-Modified)|(Location)/i'
```

Comment:
I think my regexp was poor in the first place... the `|` operator incorrectly allows you to exclude the `^` beginning of string matcher.

It needs a surrounding group to match them better. E.g. `/^((A)|(B))/i`. I prefer this to peppiring in additional `^` operators. :)